package.json

```diff
@@ -23,10 +23,13 @@
 		"eslint-plugin-promise": "^6.0.0",
 		"eslint-plugin-vue": "^8.5.0",
 		"eslint-webpack-plugin": "^3.1.1",
+		"jquery": "^3.6.0",
 		"linkifyjs": "^3.0.5",
 		"node-polyfill-webpack-plugin": "^1.1.4",
 		"postcss-html": "^1.3.0",
 		"prettier": "^2.5.1",
+		"react": "^18.0.0",
```

Comment:
Why include React?

Reply:
It's a dependency of linkifyjs

Reply:
So just to clarify, the 3 dependencies added here are all dependencies of linkifyjs

Reply:
Ok